person.rb

```diff
@@ -25,6 +25,10 @@ def correct_name
     name
   end
 
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
+
```

Comment:
I suggest the add rental method should have the book to be rented and the date as arguments and not the person😉. and the person is what should be sent as `self` to achieve the `has_many` relationship😉.

```suggestion
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
  
```

 also, Kindly add the rental to the attr_accessor since we are reading and writing to it and the rentals should be initialised in the constructor😉 like `@rental = []`.